config/example_aliases.json

```diff
@@ -1,8 +1,23 @@
 {
+    "--comment": [
+        "Aliases allow you to rename or shorten existing commands.                          ",
+        "Alias names may contain anything, except for spaces.                               ",
+        "The keys must be valid commands, and can have simple, space separated arguments.   ",
+        "Arguments passed by the user will be appended to any command arguments set here.   ",
+        "                                                                                   ",
+        "For Example:                                                                       ",
+        {
+            "_command_name_":  ["alias1", "a_long_alias"],
+            "_cmd_ arg1 arg2 argN": ["short"],
```

Comment:
This could use some clarification, maybe an example of what an argument is?

Reply:
Yeah, probably right.  Only real issue is trying to write comments in JSON, which doesn't have comments naturally.   
I'm open to suggestions on how to improve it.

Reply:
Hmm that does complicate things, I didn't know json couldn't contain comments, any chance of making it an ini file 😆

Reply:
Eh, they probably could but I don't think I'll get around to redoing the format and parsing for this PR. lol  
Will admit that this basically yet another bolt-on and not really a rework.